mta/mta_services_test.go

```diff
@@ -1426,6 +1426,28 @@ var _ = Describe("MtaServices", func() {
 			Ω(messages).Should(BeEmpty())
 		})
 
+		It("Get build parameters using extension file", func() {
+			myBuilder := ProjectBuilder{
+				Builder: "mybuilder",
+			}
+			otherBuilder := ProjectBuilder{
+				Builder: "otherbuilder",
+			}
+			oBuildParameters := ProjectBuild{
+				BeforeAll: []ProjectBuilder{myBuilder},
+				AfterAll:  []ProjectBuilder{otherBuilder},
+			}
+
+			mtaPath := getTestPath("mta.yaml")
+			extPath := getTestPath("mta.mtaext")
```

Comment:
Please add a comment that build parameters are invalid in the mtaext and we don't merge them

Reply:
added